src/main/kotlin/ro/hibyte/betting/service/WaspService.kt

```diff
@@ -0,0 +1,46 @@
+package ro.hibyte.betting.service
+
+import org.springframework.core.io.ByteArrayResource
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.util.MultiValueMap
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.multipart.MultipartFile
+import org.json.JSONArray
+
+
+@Service
+class WaspService {
+
+    private val restTemplate = RestTemplate()
+
+    fun sendPhotoToWasp(photo: MultipartFile): Long {
+
+        val headers = HttpHeaders().apply {
+            contentType = MediaType.MULTIPART_FORM_DATA
+            set("X-API-KEY", "a4c1098d-9fda-4b27-b268-62a066f0cddd")
```

Comment:
Extract the API key to a config [@Value](https://www.baeldung.com/spring-value-annotation). (let me know when you get to this)